src/sentry/migrations/0918_sentry_release_django_arrayfield.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 5.2.1 on 2025-06-03 21:04
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.new_migrations.monkey.special import SafeRunSQL
+
+# output from the original `django sqlmigrate`:
+#
+# ```
+# UPDATE "sentry_release" SET "authors" = '{}' WHERE "authors" IS NULL; SET CONSTRAINTS ALL IMMEDIATE;
+# ALTER TABLE "sentry_release" ADD CONSTRAINT "sentry_release_authors_68da3e0a_notnull" CHECK ("authors" IS NOT NULL) NOT VALID;
+# ALTER TABLE "sentry_release" VALIDATE CONSTRAINT "sentry_release_authors_68da3e0a_notnull";
+# ALTER TABLE "sentry_release" ALTER COLUMN "authors" SET NOT NULL;
+# ALTER TABLE "sentry_release" DROP CONSTRAINT "sentry_release_authors_68da3e0a_notnull";
+# ```
+#
+# the first statement is really expensive so we're skipping it -- there
+# shouldn't be nulls and the constraint will catch them if there are
```

Comment:
hmmmm this might not be true.  tried a few queries on earlier ids in redash and I'm seeing a bunch of nulls 🤔